modules/learn/pages/clusters-and-availability/xdcr-conflict-logging-feature.adoc

```diff
@@ -0,0 +1,585 @@
+
+A copy of conflict source document ID prefixed with `src_`.
+
+A copy of conflict target document ID prefixed with `tgt_`.
+
+| `docId`
+| document id of the actual document in conflict.
+
+| `replId`
+| Replication ID.
+
+| `srcDoc` and `tgtDoc`
+These JSON Objects have the IDs of the source and target document copies in the conflict collection.
+The document IDs are indicated by `srcDoc.id` and `tgtDoc.id`.
+
+| `nodeId`
+| Node identification, which can be the Hostname or IP address and a port.
+
+| `bucketUUID`
+| Source/target bucket unique ID.
+
+| `cas`
+| Source/target document CAS.
+
+| `clusterUUID`
+| Source/target cluster unique ID.
+
+| `isDeleted`
+| True or false for - is the source/target document deleted?
+
+| `scope` and `collection`
+| Name of the scope and collection where the conflict source/target documents exist.
+
+| `revSeqno`
+| Source/target document’s revision sequence number.
+
+| `expiry`
+| Source/target document’s expiry time.
+
+| `flags`
+| Source/target document’s flag.
+
+| `xattrs`
+| System xattributes.
+
+| `_vv`
+| Document version vector.
+HLV metadata.
+|===
+
+[#conflict-logger]
+=== Conflict Logger
+
+The XDCR Conflict Logger logs conflicts continuously and is designed to minimize the impact on replication performance and system resources.
+
+Conflict logging is performed on a best-effort basis.
+If system resources are limited or errors occur while writing to the conflict log collection, XDCR prioritizes replication and High Availability.
+In these cases, temporarily the conflict logging may be skipped or the conflict logs are dropped.
+
+By default, the Conflict Logger is tuned to handle a low percentage of conflicts.
+
+`xdcr_clog_status` is a Prometheus stat, which indicates the status of the Conflict Logger.
+The status can be one of the following:
+
+* *Hibernated*: The logger is temporarily disabled due to persistent errors or resource constraints.
++
+NOTE: Hibernation prevents the degradation of the replication performance. Logging is re-enabled after a set interval, or after the errors are manually resolved and the logger is manually restarted.
+
+* *Running*: The logger is actively processing conflict events.
+
+* *DNE*: The logger does not exist; conflict logging is not enabled.
+
+[#tunable-replication-settings]
+== Conflict Log Settings
+
+By default, the xref:learn:clusters-and-availability/xdcr-conflict-logging-feature.adoc#conflict-logger[Conflict Logger] is tuned to handle a low percentage of conflicts.
+
+You can retrieve the conflict logging settings for a replication using the REST API `GET /settings/replications/`.
+For more information, see xref:rest-api:rest-xdcr-adv-settings.adoc#rest-api-conflict-logging-replication-tunables[Retrieve Conflict Logging Settings].
+The conflict logger settings start with the prefix `cLog`.
+
+NOTE: Couchbase recommends that you must not adjust the conflict logger settings unless advised to do so by the Couchbase Support.
```

Comment:
Typo in the sentence (extraneous "must").
Maybe rewrite sentence.

NOTE: Couchbase recommends only modifying the conflict logger settings under Support advice.